kis_api.py

```
@@ -1,372 +1,850 @@
+class KISAPIClient:
+    def __init__(self):
+        self.logger = logging.getLogger(self.__class__.__name__)
+
+        # 기본 로거 사용 (main.py에서 설정된 것 사용)
+        self.logger.setLevel(getattr(logging, LOG_LEVEL, 'INFO'))
+
+        # TokenManager 초기화 (실현손익 API용)
+        try:
+            from token_manager import TokenManager
+            self.token_manager = TokenManager()
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)